src/GameController.cpp

```diff
@@ -47,6 +51,51 @@ GameController::GameController() :
        calculate the duration of the watching period before the beginning
        of the playing period */
     displayLevelTime = utils::Context::get().getClockMillisecondsTime();
+
+    /* update the dashboard total stars amount according to the value returned
+       by the level object */
+    dashboard.updateStarsAmountSurface(level.getStarsAmount());
+
+    /* create the grey rectangle shape that is displayed when the player loses
+       the current game */
+
+    /* the position of the filter is the left top corner as it is displayed on
+       the whole screen */
+    greyFilter.setPosition(0.f, 0.f);
+
+    /* the size of the filter is the whole window size */
+    greyFilter.setSize(
+        sf::Vector2f(
+            window::WIDTH,
+            window::HEIGHT
+        )
+    );
+
+    /* the color of the grey filter is grey with a light transparency */
+    greyFilter.setFillColor(
+        colors::ColorsManager::get().getColorPartialDarkGrey()
+    );
+
+    /* initialize the SFML text that is displayed when the player loses the
+       game */
+
+    /* the text is aligned at the center of the window */
+    loseText.setPosition(
+        400.f,
+        200.f
+    );
+
+    /* the text contains the lose message */
+    loseText.setString("You Lose !");
+
+    /* the size of the lose message is the same as the title items sizes */
+    loseText.setCharacterSize(fonts::TITLE_SIZE);
+
+    /* the font of the lose message is the normal text font */
+    loseText.setFont(fonts::FontsManager::get().getTextFont());
+
+    /* the lose text is written in red color on the grey filter */
+    loseText.setColor(colors::ColorsManager::get().getColorRed());
```

Comment:
the creation of the filter shape and the lose text must be into separated method